guides/plugins/apps/payment.md

```diff
@@ -73,7 +77,7 @@ Request content is JSON
 
 Refer to an example on [payment payload](https://github.com/shopware/app-php-sdk/blob/main/tests/Context/_fixtures/payment.json) and the response should look like this:
 
-```json
+```json5
```

Comment:
@lernhart, is there any known drawback of using json5 in markdown that does not necessarily need it? As far as I understand, the main benefit is better syntax highlighting in IDEs, for example, comments, etc.

If not, we might replace all of them docs-wide in the future.

Reply:
There is a policy that all code examples should work and be copy-pastable for external devs. 

As we use comments in our json examples, which is Not allowed with JSON, I wanted to emphasize, that these examples are working with JSON5 only, so nobody wonders why we are using comments there.